server/server.go

```diff
@@ -92,6 +92,11 @@ const (
 	idAllocLabel = "idalloc"
 
 	recoveringMarkPath = "cluster/markers/snapshot-recovering"
+
+	// PDMode represents that server is in PD mode.
+	PDMode = "PD"
+	// APIServiceMode represents that server is in API service mode.
+	APIServiceMode = "API service"
```

Comment:
It needs to support API Service Mode and TSO Service Mode at the same time if TSO service falls back to API service -- how do we represent it?

Reply:
For the original PD related starting process, we only support either PD-only mode or API service mode.

Reply:
I see